shopinvader_product_template_multi_link/tests/test_product.py

```diff
@@ -54,7 +54,9 @@ def setUpClass(cls):
         cls.shopinvader_variant_3_2 = cls.variant_3_2._get_invader_variant(
             cls.backend, "en_US"
         )
-
+        cls.link_type_sym = cls.env["product.template.link.type"].create(
```

Comment:
Replace 'cls.link_type_sym' by 'cls.link_type_asym' to be consistent with 'is_symmetric: False'

Reply:
done